package.json

```diff
@@ -12,7 +12,7 @@
     "backend:test-api": "NODE_ENV=test PGPASSFILE=/mnt/ramdisk_move_test/.pgpass API_TEST_HEADLESS=1 node  -r ./globalWindowShim -r esm -r module-alias/register web/server.js",
     "ci:jest-coverage": "NODE_ENV=test PGPASSFILE=/mnt/ramdisk_move_test/.pgpass jest --coverage",
     "ci:npm-audit": "npm audit",
-    "ci:npm-outdated": "npm outdated || true",
+    "ci:npm-outdated": "npm outdated",
```

Comment:
now that we've been _completely_ up to date on dependencies for a while, let's remove this `|| true` guard so that this is strictly checked on each release :)